### PR TITLE
ZEA-2475: Bind domain in cli

### PR DESCRIPTION
**Pull request by hackerchai**

### CODE REVIEW

File: internal/cmd/service/service.go [hunk 1]
```diff
@@ -31,6 +32,7 @@ func NewCmdService(f *cmdutil.Factory) *cobra.Command {
 	cmd.AddCommand(serviceRedeployCmd.NewCmdRedeploy(f))
 	cmd.AddCommand(serviceSuspendCmd.NewCmdSuspend(f))
 	cmd.AddCommand(serviceDeployCmd.NewCmdDeploy(f))
+	cmd.AddCommand(serviceDomainCmd.NewCmdCreateDomain(f))
```
Comment (MichaelYuhe) [inline internal/cmd/service/service.go:35]: The command seems to be registered as a sub command in domain already?

**Review by MichaelYuhe — APPROVED** [review]

File: internal/cmd/service/service.go (hunk 1 above)
Comment (hackerchai) [reply internal/cmd/service/service.go]: > The command seems to be registered as a sub command in domain already?

fixed

**Review by MichaelYuhe — APPROVED** [review]

**Review by MichaelYuhe — APPROVED** [review]
LGTM. cc @pan93412

**Review by pan93412 — COMMENTED** [review]
Improve the writing of CLI partially 🤔

**Pull request merged**